MapboxCoreNavigation/DistanceFormatter.swift

```diff
@@ -20,6 +20,49 @@ extension CLLocationDistance {
     var kilometers: CLLocationDistance {
         return self / 1000
     }
+    
+    func inMiles() -> Double {
+        return self * .metersPerMile
+    }
+    
+    func inYards() -> Double {
+        return self * .feetPerMeter / 3
```

Comment:
Nit: I would feel better about a `.yardsPerMeter` case that gets rid of the magic value, but it's pretty common knowledge that there are 3 feet in a yard. This might be me getting a little overly pedantic.